ingress/controllers/nginx-third-party/README.md

```diff
@@ -426,6 +426,16 @@ Please check the example `rc-custom-configuration.yaml`
 
 If the Configmap it is updated, NGINX will be reloaded with the new configuration
 
+### NGINX status page
+
```

Comment:
please also rename the rc so it doesn't say something like 3rdparty etc, i.e can we just call this one nginx-ingress-controller? 

Actually now that we're not dependent on third party (right?) I think we can rename the whole directory to just controllers/nginx. WDYT?